### PR TITLE
allow printing log messages during occ

### PR DESCRIPTION
**Pull request by icewind1991**
By setting `OCC_LOG=<level>` all log messages created while running the command will be printed to `stderr`.

- [x] requires https://github.com/nextcloud/server/pull/38490

### CODE REVIEW

File: lib/Log/Console.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	public function __construct(Formatter $formatter, SystemConfig $config, string $level, int $terminalWidth) {
+		parent::__construct($config);
+		$this->formatter = $formatter;
+		$this->level = self::parseLogLevel($level);
+		$this->terminalWidth = $terminalWidth;
+	}
+
+	public function log(int $level, string $app, array $entry) {
+		if ($level >= $this->level) {
+			$messageWidth = $this->terminalWidth - 8 - 18 - 6;
```
Comment (blizzz) [inline lib/Log/Console.php:47]: magic numbers (and in following lines are more magic numbers)

**Review by blizzz — APPROVED** [review]
with non-blocking comments

File: .github/workflows/phpunit-mysql.yml [hunk 2]
```diff
@@ -58,6 +58,7 @@ jobs:
 
       - name: Enable ONLY_FULL_GROUP_BY MySQL option
         run: |
+          sleep 5
```
[the remark this answers is not in the document]
Comment (icewind1991) [reply .github/workflows/phpunit-mysql.yml:61]: The job is unreliable without, as mysql isn't always fully running at this point

File: lib/Log/Console.php (hunk 1 above)
Comment (icewind1991) [reply lib/Log/Console.php:47]: They are the column widths that "seem to work best"

Same number as https://github.com/nextcloud/logreader/blob/0718de62d4963d8ffe731cd7141db0b74e6503ce/lib/Command/Tail.php#L75

Comment (blizzz) [reply lib/Log/Console.php:47]: Better kept in named constants imo.

**Pull request merged**